template/index.html

```diff
@@ -1,2 +1 @@
-
```

Comment:
this change doesn't seems necessary to me

Reply:
this is generated file, previously i used to manually add a newline to the file to pass the `config/travis/run-checks.sh`. the `no-line-end` part.

now i changed the "compiler" to print out the line-end chracter, so dun need the newline temporary hack anymore.